pkg/hope/node_management.go

```
@@ -136,11 +135,13 @@ func CreateClusterMaster(log *logrus.Entry, node *Node, podNetworkCidr string, l
 		lbMasters = append(lbMasters, aMaster)
 	}
 
-	SetLoadBalancerHosts(log, loadBalancer, &lbMasters)
+	if err := SetLoadBalancerHosts(log, loadBalancer, &lbMasters); err != nil {
```

Should probably propagate this error.

---

nah